PSI Food waste/Pages/Forms/EditProduct.cshtml.cs

```diff
@@ -65,7 +65,7 @@ public async Task<IActionResult> OnPostAsync()
                 }
             }
 
-            return RedirectToPage("./RestaurantVerified");
+            return RedirectToPage("./RestaurantVerified", "Get", new { ID = Product.RestId });
```

Comment:
Ok, so I looked up into your CreatedRestaurants and just passed the currentRestaurant id back. that fixed it.